src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/03-create-cluster-review.e2e-spec.ts

```diff
@@ -10,13 +10,12 @@ describe('Create Cluster Review page', () => {
     createCluster.createCluster();
 
     cy.get('button[aria-label="Next"]').click();
+    cy.get('button[aria-label="Next"]').click();
```

Comment:
is this duplicated or intentional?

Reply:
This is intentional. Thanks

Reply:
Btw can u rename file name as "04-..." @aaSharma14